tests/test.py

```diff
@@ -1213,6 +1213,56 @@ def test_gen_keys_with_empty_client_id(self):
             shutil.rmtree('/tmp/.testkeys')
 
 
+class KeyMakerTestWithAWSKMS(unittest.TestCase):
+    def setUp(self):
+        if not TEST_WITH_AWS_KMS:
+            self.skipTest("test with AWS KMS ACRA_MASTER_KEY loader")
+
+        self.kms_client = AWSKMSClient()
+        self.create_configuration_file()
+
+    def create_configuration_file(self):
+        configuration = {
+            'access_key_id': 'access_key_id',
+            'secret_access_key': 'secret_key_id',
+            'region': 'eu-west-1',
+            'endpoint':  os.environ.get('AWS_KMS_ADDRESS', 'http://localhost:8080')
+        }
+        cfg_json_string = json.dumps(configuration)
+        self.config_file = tempfile.NamedTemporaryFile('w+', encoding='utf-8').name
+
+        # Open the file for writing.
+        with open(self.config_file, 'w') as f:
+            f.write(cfg_json_string)
+
+    def test_generate_master_key_with_kms_create(self):
+        master_key_file = tempfile.NamedTemporaryFile('w+', encoding='utf-8')
+        subprocess.check_output(
+            [os.path.join(BINARY_OUTPUT_FOLDER, 'acra-keymaker'), '--keystore={}'.format(KEYSTORE_VERSION),
+            '--generate_master_key={}'.format(master_key_file.name),
+            '--kms_type=aws',
+            '--kms_credentials_path={}'.format(self.config_file)])
+
+        resp = self.kms_client.list_aliases()
+        created_arn = resp['Aliases'][0]['AliasArn']
+
+        ciphertext = open(master_key_file.name, "rb").read()
+        decrypt_resp = self.kms_client.decrypt(keyId=created_arn, ciphertextBlob=ciphertext)
+        self.assertEqual(len(decrypt_resp['Plaintext']), 32)
```

Comment:
lets check that plaintext not equal to ciphertext too?

Reply:
Sure